### ISSUE TITLE
Enable country-level labels for disaggregation types

### ISSUE THREAD

**Issue by tolabot** [toladata/TolaActivity#247]
gfox@mercycorps.org On admin and UI sides, will be more organized/ clear if disaggregation labels can be assigned per country level . Eg, multiple countries may use Governorate or Province as a standard disaggregation type, but will have unique labels that apply only to their country instance.      # - On admin and UI sides, will be more organized/ clear if disaggregation labels can be assigned per country level . Eg, multiple countries may use Governorate or Province as a standard disaggregation type, but will have unique labels that apply only to their country instance. <br><br>KSANDSTROM@MERCYCORPS.ORG added a comment   - Waiting on standard disaggregation decision<br>None<br>KSANDSTROM@MERCYCORPS.ORG changed ticket status from [Open - On Hold] to [ Resolved]<br>
